tools/devops/Makefile

```diff
@@ -0,0 +1,10 @@
+TOP=../..
+include $(TOP)/Make.config
+
+xcode-deps.csx: xcode-deps.csx.in Makefile $(TOP)/Make.config
```

Comment:
You're not only installing Xcode, but several other dependencies as well, which makes the filename misleading. What about something like `device-tests.csx` instead?

Reply:
`device-tests-provisioning.csx` Would make it a little more explicit the intent